packages/riipen-ui/src/utils/container.js

```diff
@@ -0,0 +1,10 @@
+export function getDocument(node) {
```

Comment:
For all these util functions, maybe write a JSDoc style comment above them describe what they do, their arguments, their return value, etc.

Reply:
👍